cmd/dagger/flags.go

```diff
@@ -290,7 +290,10 @@ func (v *secretValue) Get(ctx context.Context, c *dagger.Client) (any, error) {
 	case envSecretSource:
 		envPlaintext, ok := os.LookupEnv(v.sourceVal)
 		if !ok {
-			return nil, fmt.Errorf("secret env var not found %q", v.sourceVal)
+			// Don't show the entire env var name, in case the user accidentally passed the value instead...
+			// This is important because users originally *did* have to pass the value, before we changed to
+			// passing by name instead.
+			return nil, fmt.Errorf("secret env var not found %q...", v.sourceVal[:3])
```

Comment:
This will panic if `v.sourceVal` is less than 3 chars (dumb corner case, but I guess `env:CI` is maybe a real world example). Can just check it's length and truncate it to at most 3.

Reply:
ChatGPT, you had one job...

<img width="595" alt="Screenshot 2024-01-12 at 18 23 32" src="https://github.com/dagger/dagger/assets/29565/27d5c8b5-6ffc-4388-a7c8-0e9618bb150e">

Reply:
I think that was my most intellectually lazy use of ChatGPT ever...